src/clients/c++/perf_client.cc

```diff
@@ -342,6 +343,11 @@ Usage(char** argv, const std::string& msg = std::string())
       << " of latency will also be reported and used to detemine if the"
       << " measurement is stable instead of average latency."
       << " Default is -1 to indicate no percentile will be used." << std::endl;
+  std::cerr << "For --shape, it set the shape used for the specified input. The"
+               " argument must be specified as 'name:shape' where the shape is"
```

Comment:
.. where the shape is a comma-separate list of dimension sizes, for example --shape input:1,2,3.